ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/RpcErrorType.java

```diff
@@ -24,6 +24,7 @@ public enum RpcErrorType {
   TIMEOUT_ERROR(-32603, "Timeout expired"),
 
   METHOD_NOT_ENABLED(-32604, "Method not enabled"),
+  UNSUPPORTED_FORK(-38005, "Unsupported fork"),
```

Comment:
you might find this is duplicated when you update this branch...(https://github.com/hyperledger/besu/blob/main/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/RpcErrorType.java#L84)